examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+kind: Gateway
+metadata:
+  name: udp-gateway
+  namespace: stunner
+spec:
+  gatewayClassName: stunner-gatewayclass
+  listeners:
+    - name: udp-listener
+      port: 3478
+      protocol: UDP
+EOF
+```
+
+This will create us a whole new service:
+
+```console
+kubectl get services -n stunner
+
+NAME                              TYPE           CLUSTER-IP      EXTERNAL-IP                  PORT(S)          AGE
+stunner                           LoadBalancer   10.85.128.58    <STUNNER_EXTERNAL_IP>        3478:31362/UDP   13m
+stunner-gateway-udp-gateway-svc   LoadBalancer   10.85.131.128   <GATEWAY_EXTERNAL_IP>        3478:32576/UDP   30s
+```
+
+
+Note down <GATEWAY_EXTERNAL_IP> and the port assigned to it (if not using NodePort, 3478 in our case). This will be the address the clients will need to connect to for the TURN server.
+
+
+Finally, attach a UDP route to the Gateway, so that clients will be able to connect via the public STUN/TURN listener UDP:3478 to the worker LoadBalancer service we`ve created.
+In our case, we named it worker-ci-udp-svc. Don`t forget to specify the namespace, even if its in the default one.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: UDPRoute
+metadata:
+  name: worker-udp-route
+  namespace: stunner
+spec:
+  parentRefs:
+    - name: udp-gateway
+  rules:
+    - backendRefs:
+        - name: worker-ci-udp-svc
+          namespace: cloudretro
+EOF
+```
+
+To make sure everything is set, you can check if the stunnerd-configmap includes your workers pod IPs as endpoints. If it does, you are almost all set!
+
+```console
+kubectl get configmap -n stunner stunnerd-config -o yaml
+```
+
+#### Last touches
+
+Before we can establish an RTCP connection, we will need to configure our CloudRetro pods once again.
+To be specific, we need the coordinator to send the TURN server address to the client:
+
+```console
+kubectl edit configmap -n cloudretro cloudretro-config
+```
+
+Here, we are looking for the attribute iceServers: -> url.
+To make it work, you should replace:
+
+* stun:stun.l.google.com:19302 (default value) with turn:<GATEWAY_EXTERNAL_IP>:PORT (what you`ve hopefully noted down in the previous step).
+
+This unfortunately not enough all by itself, you will need to add two extra attributes to the config files, right under the url one.
+Change the username and the credential according to what you`ve specified at the GatewayConfig.
+
+* username: user-1
+* credential: pass-1
+
+After saving, you should restart the coordinator deployment to make changes take effect:
+
+```console
+kubectl rollout restart deployment -n cloudretro coordinator-deployment
+```
+
+And there You go! After all set, connecting to the coordinator will... have no change. Seemingly.
+But if You open the browser console, You should be seeing a marvelous "[rtcp] connected" log message.
+Why doesn`t it work then?
+It has no games.
+That`s work under progress.
+Hard times over anyway.
```

Comment:
Please include the standard footer we use in all examples (see everything that comes after this line: https://github.com/l7mp/stunner/tree/main/examples/direct-one2one-call#clean-up). you can just copy-paste the README.md from the existing examples, but please include a Clean up section so that users can remove al objects they have created here after they've finished playing with cludretro